README.md

```diff
@@ -672,4 +672,18 @@ When running in insecure mode, the secrets are retrieved from the *Writable.Inse
 
 #### Storing Secrets
 
-<placeholder>
\ No newline at end of file
+Secrets for an application service can be stored in the secret store (Vault) by making an HTTP `POST` call to `http://[host]:[port]/secrets`.  
```

Comment:
Add something about only valid in running in secure mode and for insecure mode the insecure secrets can be added to config or in consul.